tests/unit/valgrind.suppressions

```diff
@@ -10,3 +10,14 @@
    fun:pthread_create@@GLIBC_2.2.5
    fun:main
 }
+
+# Memory is released by s2n_map_free when s2n_cleanup is called
```

Comment:
Hm, why does LSAN and Valgrind complain?

Reply:
> Hm, why does LSAN and Valgrind complain?

That is good catch. I have looked into this, init table is called twice. one by BEGIN_TEST(), another one by API call. I have added checking to prevent double init and free :)